blockchain/evm.go

```diff
@@ -54,7 +54,13 @@ func NewEVMContext(msg Message, header *types.Header, chain ChainContext, author
 	} else {
 		beneficiary = *author
 	}
-	baseFee = new(big.Int).SetUint64(params.BaseFee)
+	// TODO-klaytn kip71 fork
+	// bafore fork, base fee is 0
```

Comment:
Do we need an additional code handling hard-fork block number? `header.BaseFee` exist only when kip71 hard-fork enabled, so we don't need to check again whether kip71 hard-fork enabled or not.

Reply:
Could you propose example `code snippet`? I think we need to check the hardfork because there has been `header.BaseFee` already. Therefore we can manage BaseFee opcode and API before hardfork.

Reply:
I refactored to remove redundant kip71 hardfork conditions. @aidan-kwon Please review again.